src/aiida/cmdline/commands/cmd_storage.py

```diff
@@ -184,6 +187,11 @@ def storage_maintain(ctx, full, no_repack, force, dry_run, compress):
     if not dry_run and not force and not click.confirm('Are you sure you want continue in this mode?'):
         return
 
+    if STORAGE_LOGGER.level <= logging.REPORT:
+        set_progress_bar_tqdm(leave=STORAGE_LOGGER.level <= logging.INFO)
```

Comment:
Not important for this PR, but spend some time to understand this.

One was very specific about the behavior of the bar, choosing logging.INFO instead of logging.REPORT. The behavior is only understandable when reading the tqdm default arguments and it is very particular. But makes actually sense. One could have put a comment here like 
```suggestion
        # Only keep the first tqdm bar if it is info. To keep the nested bar information one needs report level
        set_progress_bar_tqdm(leave=STORAGE_LOGGER.level <= logging.INFO)
```